tests/cmd/stability/main.go

```diff
@@ -229,12 +208,6 @@ func run() {
 		oa.BackupAndRestoreToMultipleClustersOrDie(clusters[0], backupTargets)
 		klog.Infof("clusters backup and restore checked")
 
-		// delete operator
-		oa.CleanOperatorOrDie(ocfg)
-		oa.CheckOperatorDownOrDie(deployedClusters)
-		oa.DeployOperatorOrDie(ocfg)
-		klog.Infof("clusters operator deleted and redeployed, checked")
```

Comment:
this has been covered by [this test](https://github.com/pingcap/tidb-operator/blob/cd00a2079ba89a34909c0ca41ee20aa62259267c/tests/e2e/tidbcluster/stability.go#L141-L146) which verifies that tidb cluster should not be affected if the operator is deleted